.eslintignore

```diff
@@ -2,3 +2,4 @@
 public
 docs/
 meetings/
+.github
```

Comment:
Don't think there is any JS files that ESLint would need to ignore. This would also add back the missing EOF
```suggestion

```

Reply:
oh yeah!